### DISCUSSION TITLE
Gaps between windows?

### DISCUSSION THREAD

**Discussion by muhammad-syahrizal** [i3/i3#4518] [Q&A] [answered]
Are there any plans to support some kind of gaps in between windows? Is it out of question? If it's already implemented, how can I make use of that? Thank you.

**Comment by Airblader** [comment, marked as answer]
You're looking for [i3-gaps](https://github.com/Airblader/i3). Otherwise see https://github.com/i3/i3/issues/3724.

**Reply by slith76** [reply]
Is there a plan to integrate your changes regarding gabs into the official repository? 🤔

**Reply by Airblader** [reply]
That'd be #3724.
